src/raylib.h

```diff
@@ -1394,6 +1395,7 @@ RLAPI void ImageDrawTextEx(Image *dst, Font font, const char *text, Vector2 posi
 // NOTE: These functions require GPU access
 RLAPI Texture2D LoadTexture(const char *fileName);                                                       // Load texture from file into GPU memory (VRAM)
 RLAPI Texture2D LoadTextureFromImage(Image image);                                                       // Load texture from image data
+RLAPI Texture2D LoadTextureMaskFromImage(Image image, Color color, float threshold);                            // Load texture mask from image data (alpha channel)
```

Comment:
I don't think this function is actually needed. Just with `ImageFromChannel()` + `LoadTextureFromImage()` the same result could be acomplished. I prefer to minimize raylib functions.